### ISSUE TITLE
Before/After Events for Element Integration

### ISSUE THREAD

**Issue by strandofgenius** [verbb/formie#532]
Ability to call before/after event listeners for element integrations.

**Comment by engram-design** [comment]
Should be fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.10",
  "...": "..."
}
```

Then run `composer update`.

You'll be able to use the [beforeSendPayload](https://verbb.io/craft-plugins/formie/docs/developers/events#the-beforeSendPayload-event) event or the [afterSendPayload](https://verbb.io/craft-plugins/formie/docs/developers/events#the-afterSendPayload-event) event

**Comment by engram-design** [comment]
Fixed in [1.4.11](https://github.com/verbb/formie/releases/tag/1.4.11)

**Comment by strandofgenius** [comment]
Awesome! Thank you.
